docs/topics/gradle.md

```diff
@@ -256,13 +256,12 @@ kotlin.stdlib.default.dependency=false
 
 ### Set dependencies on test libraries
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
-
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing Kotlin projects on different Kotlin platforms.
+Add the dependency `kotlin-test` to the `commonTest` source set, and the Gradle plugin will infer the corresponding 
+test dependencies for each source set:
```

Comment:
Do we mean test source sets here? If we do, then the current wording is not completely true.
If we really mean _each_ source set in the project, then the behavior is strange =)

Reply:
Just to recheck:
@ilya-g, does Gradle infer the corresponding test dependencies for each **test** source set?

Reply:
It depends on where `kotlin("test")` dependency is added. If in `commonTest`, then all test source sets actualizing `commonTest` also get the respective variant of `kotlin-test` dependency.